### PR TITLE
[12.0][sql_export] Fix uses of params in sql query

### PR DESCRIPTION
**Pull request by florian-dacosta**
Same as https://github.com/OCA/server-tools/pull/1613
It fixes the use of params in sql_export module.

@bguillot @bealdav @mreficent @legalsylvain

### CODE REVIEW

File: sql_export/tests/test_sql_query.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import base64
 from odoo.tests.common import TransactionCase, post_install
 from odoo.exceptions import Warning as UserError
```
Comment (bealdav) [inline sql_export/tests/test_sql_query.py]: please remove Warning

File: sql_export/tests/test_sql_query.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
...
+from openerp import fields
```
Comment (bealdav) [inline sql_export/tests/test_sql_query.py]: s/openerp/odoo

**Review by bealdav — COMMENTED** [review]
nitpicking

**Review by bealdav — APPROVED** [review]
Tested with runbot.

Works with date

**Comment by florian-dacosta** [comment]
Changes done, thanks

**Review by kevinkhao — APPROVED** [review]
Functional test:
LGTM

**Pull request merged**